src/PathFinderUtils.js

```diff
@@ -0,0 +1,172 @@
+import Dongle from './Dongle'
+import Networks from './Networks'
+import bitcoin from 'bitcoinjs-lib'
+import bs58 from 'bs58'
+import _ from 'lodash'
```

Comment:
good practice for lodash is to import directly the methods to lower the bundle size. like `import padStart from "lodash/padStart"`